kolibri/core/content/management/commands/importchannel.py

```diff
@@ -120,21 +120,22 @@ def _start_file_transfer(self, filetransfer, channel_id, dest):
                         filetransfer.cancel()
                         break
                     progress_update(len(chunk), progress_extra_data)
-                try:
-                    import_channel_by_id(channel_id, self.is_cancelled)
-                except channel_import.ImportCancelError:
-                    # This will only occur if is_cancelled is True.
-                    pass
                 if self.is_cancelled():
                     try:
                         os.remove(dest)
-                    except IOError as e:
+                    except OSError as e:
                         logger.error(
                             "Tried to remove {}, but exception {} occurred.".format(
                                 dest, e
                             )
                         )
                     self.cancel()
```

Comment:
another thing I notice is that during this cancel() call, an exception will be raised, which will be caught in line 141. probably from iceqube?
so in the log file, it has
```
ERROR    An error occurred during channel import: 
```
I wonder if we can add `return True` after the cancel() call.

Reply:
I am not actually sure what the point of that cancel call is, except to cause an error, so I have just deleted it.